docs/content/modeling/roles-and-permissions.mdx

```diff
@@ -219,16 +218,16 @@ To allow `viewers` of a `trip` to have **permissions to view bookings** and `own
   }}
 />
 
-> Note: notice how both `booking_viewer` and `booking_adder` don't have [direct relationship type restrictions](../configuration-language.mdx#the-direct-relationship-type-restrictions). This is to ensure that the relation can only be assigned through the **role** and not directly.
+> Note: both `booking_viewer` and `booking_adder` don't have [direct relationship type restrictions](../configuration-language.mdx#the-direct-relationship-type-restrictions). This ensures that the relation can only be assigned through the role and not directly.
 
 ### 03. Checking User Roles And Their Permissions
 
```

Comment:
https://oktawiki.atlassian.net/wiki/spaces/DOCS/pages/2544472397/Document+Structure#Capitalization

Reply:
Per @rhamzeh, if we would like to reference a generalized "Best practices" ... I would refer here:

https://developers.google.com/style/headings?hl=en#heading-and-title-format

Essentially, we should be using sentence case for headers. Let me know if you have any questions. Thanks!